cli/cmd/export/export.go

```diff
@@ -82,6 +93,60 @@ func (e *Export) Run(ctx *clicontext.CLIContext) error {
 	return nil
 }
 
+func collectObjectsFromStack(ctx *clicontext.CLIContext, stack *riov1.Stack) ([]runtime.Object, error) {
+	var objects []runtime.Object
+	client, err := dynamic.NewForConfig(ctx.Config.RestConfig)
+	if err != nil {
+		return nil, err
+	}
+	for _, gvk := range stack.Spec.AdditionalGroupVersionKinds {
+		gvr := schema.GroupVersionResource{
+			Group:    gvk.Group,
+			Version:  gvk.Version,
+			Resource: strings.ToLower(name.GuessPluralName(gvk.Kind)),
+		}
+		resourceClient := client.Resource(gvr)
+		ul, err := resourceClient.List(metav1.ListOptions{
```

Comment:
You should be able to list only resources in the current namespace since we scope stacks to namespaces